cmd/node_ctrl/main.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"golang.org/x/exp/maps"
+
+	"github.com/eagraf/habitat-new/internal/node/constants"
+	"github.com/eagraf/habitat-new/internal/node/controller"
+	"github.com/urfave/cli/v2"
+)
+
+var port string
+
+func printResponse(res *http.Response) error {
+	slurp, err := io.ReadAll(res.Body)
+	if err != nil {
+		return err
+	}
+
+	fmt.Printf("Status: %s\nResponse: %s\n", res.Status, string(slurp))
```

Comment:
It'd be really nice if we just output the raw JSON, that way this would work with tools like `jq` out of the box.  We could either add a flag to print just JSON output, or wrap the response JSON with a parent JSON that has the response status. Also, any non-200 response status should reflect in an exit code of 1.

Reply:
I think it's useful to see non-200 response + body in case we decide to return some info in the result.
Non-200 doesn't necessarily mean an error in the CLI program, it could be the right response to return given certain inputs